task/BuildOTPGraph.js

```diff
@@ -21,7 +25,13 @@ const buildGraph = function (router) {
     );
     const commit = version.toString().match(/commit: ([0-9a-f]+)/)[1];
 
-    const command = `docker run -e JAVA_OPTS="${JAVA_OPTS}" -v ${dataDir}/build/${router.id}:/var/opentripplanner --mount type=bind,source=${dataDir}/../logback-include-extensions.xml,target=/logback-include-extensions.xml ${dockerImage} --build --save`;
+    let command = `docker run -e JAVA_OPTS="${JAVA_OPTS}" -v ${dataDir}/build/${router.id}:/var/opentripplanner --mount type=bind,source=${dataDir}/../logback-include-extensions.xml,target=/logback-include-extensions.xml ${dockerImage} --build --save`;
+    if (ONLY_BUILD_STREET_GRAPH) {
+      command = `docker run -e JAVA_OPTS="${JAVA_OPTS}" -v ${dataDir}/build/${router.id}:/var/opentripplanner --mount type=bind,source=${dataDir}/../logback-include-extensions.xml,target=/logback-include-extensions.xml ${dockerImage} --buildStreet --save`;
+    } else if (USE_PREBUILT_STREET_GRAPH) {
+      command = `docker run -e JAVA_OPTS="${JAVA_OPTS}" -v ${dataDir}/build/${router.id}:/var/opentripplanner --mount type=bind,source=${dataDir}/../logback-include-extensions.xml,target=/logback-include-extensions.xml ${dockerImage} --loadStreet --save`;
+    }
```

Comment:
This can be then converted into a switch if we use an enum. It's slightly easier to read if command doesn't have a default value that gets reassigned.

Reply:
I created a default case for a switch statement here